requirements.txt

```diff
@@ -1,4 +1,5 @@
 matplotlib
 numpy
 pocketsphinx
+pvporcupine
```

Comment:
I think we should probably pin the versions of `pocketsphinx` and `pvporcupine` that we use in the benchmark. Snowboy uses a submodule, which pins it to a version by default